web/themes/joinup/templates/parts/header.html.twig

```diff
@@ -22,3 +24,10 @@
 
   </div>
 </div>
+<div class="mdl-layout__drawer">
+  <nav class="mdl-navigation">
```

Comment:
This menu needs to become a real menu at some point, can you add a todo and link to the related ticket? If none is there yet, please create one.